pkg/rest/config_upload.go

```diff
@@ -188,13 +188,21 @@ func getObjectIdIfAlreadyExists(client *http.Client, api api.Api, url string, ob
 		return "", err
 	}
 
+	var found = false
+	var configName = ""
 	for i := 0; i < len(values); i++ {
 		value := values[i]
 		if value.Name == objectName {
-			return value.Id, nil
+			if found {
+				util.Log.Warn("\t\t\tFound two or more configs having same name: %s. Please delete duplicates.", objectName)
```

Comment:
wondering if we'd have a benefit of skipping the early exit and actually counting the duplicates?
as is only the error case will exit earlier, and the normal case of no duplicates would iterate everything anyway

Reply:
The idea was to save few ms there as it's not really a difference if there are 2 or 10 copies of same resource and also to print warning only once. If you think we should add counter, can do this. Haven't thought it would be useful as it's always the same resource, same warning over and over again?

Reply:
Counter is now added.